src/content/1.7/development/components/_index.md

```diff
@@ -7,5 +7,7 @@ weight: 50
 # Components
 
 * [Grid Component][grid-component]
+* [Faceted Search Component][faceted-search-component]
```

Comment:
it would be nice to separate FO and BO components, maybe with labels or separate sections, like FO components list and BO components list. :)